feat: huge changes (#24)
lidofinance/state-mate: README.md

### ABIs

All required ABIs are located in the same directory as the config and placed under `abi` folder being downloaded upon the first launch. See [configs](/configs/).

> Use the --abi option to download new ABIs or compare them with existing one

Reviewer: ```suggestion
Use the --update-abi option to download all ABIs overwriting existing files 
```

### For developers

> Use the --schemas option to generate new JSON schemas in the 'schemas' folder

Author: > on yarn build with a separate .ts entry point

Wdym?
